src/texts.js

```diff
@@ -1,9 +1,20 @@
-import { readFileSync } from 'fs'
+import { readFileSync, readFile } from 'fs'
+import lodash from 'lodash'
 
-export function loadText(message, lang) {
-  return readFileSync(`messages/${message}.${lang}.txt`).toString()
+async function loadTextAsync(path, lang, name) {
+  return new Promise((resolve, reject) => {
+    readFile(path, (err, data) => {
+      if (err != null) {
+        reject(err)
+      } else {
+        resolve({[lang]: {[name]: data.toString()}})
```

Comment:
I feel like this doesn't correspond to the name of the function. `loadTextAsync` sounds like a simple file reader, but it returns data in a specific format. I'd consider extracting this formatting outside of the function, or at least change the name of the function somehow more self-explanatory. If you'll extract it, you can also consider [util.promisify()](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original)

Reply:
I renamed the function to `loadTextMessageAsync`